rpm/c-sample.spec

```diff
@@ -0,0 +1,28 @@
+Name:       c-sample
+Summary:    Sample app written in C
+Version:    0
+Release:    1
+License:    BSD
+URL:        https://www.github.com/sailfish/c-sample
+Source0:    %{name}-%{version}.tar.bz2
+BuildRequires: pkgconfig(ssu-sysinfo)
+
+%description
+Simple app written in C
+
+%prep
+%autosetup -p1 -n %{name}-%{version}
+
+%build
+cd src
+gcc -o c-sample c-sample.c -lssusysinfo
```

Comment:
I'd actually take both --clags and --libs from the ssu-sysinfo pkgconfig.

> gcc -o c-sample c-sample.c $(pkg-config --cflags --libs ssu-sysinfo)

Reply:
> I'd actually take both --clags and --libs from the ssu-sysinfo pkgconfig.
> 
> > gcc -o c-sample c-sample.c $(pkg-config --cflags --libs ssu-sysinfo)

I'll make that part of the Makefile PR, if that's ok with you?

Reply:
Let this be an intentional mistake for few days :-)